README.md

```diff
@@ -1,23 +1,145 @@
-# Ziti C SDK
+# OpenZiti C SDK
+![Build Status](https://github.com/openziti/ziti-sdk-c/actions/workflows/cmake.yml/badge.svg?branch=main)
 
-The Ziti C SDK delivers a set of functionality allowing developers to create applications utilizing a Ziti Network
-quickly and easily.
+The OpenZiti C SDK allows developers to create their own custom OpenZiti network endpoint applications.
 
-<h2>What Is Ziti</h2>
+OpenZiti is a modern, programmable network overlay with associated edge components, for application-embedded,
+zero trust network connectivity, written by developers for developers.
+The SDK harnesses that power via APIs that allow developers to imagine and develop solutions beyond what
+OpenZiti handles by default.
 
-Ziti is the name of NetFoundry's open source network overlay. The goal of Ziti is to
-reinvent networking by providing secure, scalable connectivity as a network overlay over
-existing underlay networks.
+This SDK does the following:
 
-<h2>Status</h2>
+- enable network endpoint clients allow a device to [dial (access)](#dialing-a-service)
+  or [bind (host)](#binding-to-a-service) OpenZiti Services
+- provides [authentication](https://openziti.io/docs/learn/core-concepts/security/authentication/auth) interfaces for
+  [x509 certificates](#example-code-configuration) flows
+- collects and submits security posture collection/submission
+  for [Posture Checks](https://openziti.io/docs/learn/core-concepts/security/authorization/posture-checks)
+- allows applications to bind or dial services via standard [`socket`](#high-level-zitilib-api) interfaces
```

Comment:
provides e2ee?

Reply:
I am not sure, e2ee is a feature of ziti. saying `provides e2ee` make it sound like we allow an application to hook into it